packages/dtos/src/user.ts

```diff
@@ -26,3 +27,33 @@ export class GetUserDto implements Partial<User & Casting> {
   @ApiProperty()
   userId: number
 }
+
+export class UpdateUserStatusDto {
+  @IsNotEmpty()
+  @IsEnum(UserStatus)
+  @ApiProperty({enum: UserStatus})
+  status: UserStatus
+
+  @IsOptional()
+  @IsString()
+  @ApiPropertyOptional()
+  rejectedReason?: string
+}
+
+export class PendingUserDto {
+  @ApiProperty()
+  type: UserType
+
+  @ApiProperty()
+  data: {
+    userId: number;
+    firstName: string;
+    middleName?: string;
+    lastName: string;
+    companyName?: string;
+    companyId?: string;
+    employmentCertUrl?: string;
+    idCardImageUrl?: string;
+    ssn?: string;
+  }
```

Comment:
(Question)
Do we have any concern that we cannot assign directly as property in class ?

Reply:
should I do like nested Dto instead?
`export class GetJobCardDto extends OmitType(EditJobDto, [
  'role',
  'minAge',
  'maxAge',
  'shooting',
]) {
  @ApiProperty()
  jobId: number

  @ApiProperty()
  companyName: string

  @ApiProperty()
  jobCastingImageUrl: string

  @ApiProperty()
  castingId: number

  @ApiProperty()
  castingName: string

  @ApiProperty()
  status: JobStatus
}

export class GetJobCardWithMaxPageDto {
  @ApiProperty({ type: GetJobCardDto, isArray: true })
  jobs: GetJobCardDto[]

  @ApiProperty()
  maxPage: number
}`

Reply:
I think nested dto is better